vnext/Microsoft.ReactNative.Cxx/Microsoft.ReactNative.Cxx.vcxitems

```diff
@@ -42,6 +42,7 @@
     <ClInclude Include="$(MSBuildThisFileDirectory)UI.Input.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)UI.Popups.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)UI.Text.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)XamlHelpers.h" />
```

Comment:
>)XamlHelpers.h [](start = 50, length = 14)

Is this intentional? The change appears to be adding a XamlUtils.h file.

Reply:
No, it is another artifact of failed .filters file handling...

---
In reply to: [433448516](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433448516) [](ancestors = 433448516)